src/theme/components/card.ts

```diff
@@ -164,10 +164,14 @@ const CardImage = {
           order: "2",
         }
       : {};
-    return {
-      ...imageAtEndStyles,
-      ...layoutStyles,
-    };
+    return imageAspectRatio === "original"
+      ? {
+          marginBottom: "xs",
```

Comment:
Sorry, I did not correctly specify that this bottom margin only needs to be applied for `ImageRatios.Original` in the `column` layout.  The image in the `row` layout does not need a bottom margin.